tests/integration_tests/functional/test_pause_resume.py

```diff
@@ -1,65 +1,111 @@
+    tap = microvm.ssh_network_config(context.custom['network_config'], '1')
 
     # Pausing the microVM before being started is not allowed.
-    response = test_microvm.vm.patch(state='Paused')
-    assert test_microvm.api_session.is_status_bad_request(response.status_code)
+    response = microvm.vm.patch(state='Paused')
+    assert microvm.api_session.is_status_bad_request(response.status_code)
 
     # Resuming the microVM before being started is also not allowed.
-    response = test_microvm.vm.patch(state='Resumed')
-    assert test_microvm.api_session.is_status_bad_request(response.status_code)
+    response = microvm.vm.patch(state='Resumed')
+    assert microvm.api_session.is_status_bad_request(response.status_code)
 
-    # Start microVM.
-    test_microvm.start()
+    microvm.start()
 
-    ssh_connection = net_tools.SSHConnection(test_microvm.ssh_config)
+    ssh_connection = net_tools.SSHConnection(microvm.ssh_config)
 
     # Verify guest is active.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code == 0
 
     # Pausing the microVM after it's been started is successful.
-    response = test_microvm.vm.patch(state='Paused')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Paused')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Verify guest is no longer active.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code != 0
 
     # Pausing the microVM when it is already `Paused` is allowed
     # (microVM remains in `Paused` state).
-    response = test_microvm.vm.patch(state='Paused')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Paused')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Resuming the microVM is successful.
-    response = test_microvm.vm.patch(state='Resumed')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Resumed')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Verify guest is active again.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code == 0
 
     # Resuming the microVM when it is already `Resumed` is allowed
     # (microVM remains in the running state).
-    response = test_microvm.vm.patch(state='Resumed')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Resumed')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Verify guest is still active.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code == 0
+
+    microvm.kill()
+    del tap
+
+
+@pytest.mark.skipif(
+    platform.machine() != "x86_64",
+    reason="Not supported yet."
```

Comment:
Pause/resume is available on ARM also.

Reply:
They work indeed, but have no real usecase by themselves.